lib/matplotlib/backend_bases.py

```diff
@@ -527,12 +527,12 @@ def draw_image(self, gc, x, y, im):
         """
         raise NotImplementedError
 
-    def option_image_nocomposite(self):
+    def option_combine_images(self):
```

Comment:
I did not notice this before, when did this change go in? :-1: on this and and inverting the logic.